src/wyzeapy/wyze_auth_lib.py

```diff
@@ -87,22 +112,26 @@ async def create(cls, username=None, password=None, token: Token = None, token_c
 
         return self
 
-    async def get_token_with_username_password(self, username, password) -> Token:
+    async def get_token_with_username_password(
+        self, username, password, key_id, api_key
+    ) -> Token:
         self._username = username
         self._password = create_password(password)
-        login_payload = {
-            "email": self._username,
-            "password": self._password
-        }
+        self._key_id = key_id
+        self._api_key = api_key
+        login_payload = {"email": self._username, "password": self._password}
 
         headers = {
-            'Phone-Id': PHONE_ID,
-            'User-Agent': APP_INFO,
-            'X-API-Key': API_KEY,
+            "keyid": key_id,
+            "apikey": api_key,
+            "User-Agent": "wyzeapy",
```

Comment:
I think this is a reasonable suggestion. @mrlt8 can you make this adjustment?

Reply:
This header is used only once during auth so not sure if it's really necessary as the rest of the requests are untouched and still using the existing methods/headers.

I'm also not very familiar with this codebase and unable to find any documentation, so it might be more appropriate for someone who actively uses the library to add extra configs that better suit the way other third-party apps are using it.

Reply:
I think it could be kept as APP_INFO like it was before for now, right?